README.md

```diff
@@ -79,6 +79,13 @@ h, err := libp2p.New(
 )
 ```
 
+To use TCP transport options, pass them to the `libp2p.Transport` constructor:
+```go
+h, err := libp2p.New(
+    libp2p.Transport(tcp.NewTCPTransport, tcp.DisableReuseport(), tcp.WithConnectionTimeout(20*time.Second))
```

Comment:
This seams like a good way to do this, but it's not _currently_ supported. Are you planning on implementing it?

Reply:
As of https://github.com/libp2p/go-libp2p/pull/1205, it is implemented. It's just not _released_ yet

Reply:
We probably should hold this PR back until we've cut the next go-libp2p release.

Reply:
Oh, that's really cool!